src/resources/postcss/base/full/typography/_anchors.pcss

```diff
@@ -52,7 +52,7 @@
 		 * ------------------------------------------------------------------------- */
 
 		.tribe-theme-enfold &,
-		.theme-enfold & {
+		.main_color .sidebar & {
```

Comment:
Same question here (although I haven't got Enfold up locally to test/screenshot)

Reply:
same as above